### PR TITLE
feat: add tailwind config to exports

### PR DESCRIPTION
**Pull request by jonasbuechel**

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default defineConfig({
         sourcemap: true,
         minify: true,
         rollupOptions: {
-            input: [...componentsPath.map((element) => element.path), "src/styles.css"],
+            input: [...componentsPath.map((element) => element.path), "src/styles.css", "tailwind.config.js"],
```
Comment (SamuelAlev) [inline vite.config.ts:46]: Probably not needed as it is already a `.js` file and use `require`, wdyt?

Comment (jonasbuechel) [reply vite.config.ts:46]: I added it to ensure `tailwind.config.js` is copied to `/dist` is there another way to achieve it?

Comment (SamuelAlev) [reply vite.config.ts:46]: Make sense 👍

**Review by deniaz — APPROVED** [review]

File: vite.config.ts (hunk 1 above)
Comment (jonasbuechel) [reply vite.config.ts:46]: Thanks for double-checking 🙌

**Pull request merged**